app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -79,6 +81,19 @@ object TestHelper {
         )
     }
 
+    fun verifyUrl(urlSubstring: String, resourceName: String, resId: Int) {
+        waitUntilObjectIsFound(resourceName)
+        onView(withId(resId)).check(
+            ViewAssertions.matches(
+                withText(
+                    CoreMatchers.containsString(
+                        urlSubstring
```

Comment:
Can we clean this up a bit? I don't think we need this many newlines for this method. The linter doesn't seem to complain if I just make it one line so I'd recommend doing that:
`onView(withId(resId)).check(ViewAssertions.matches(withText(CoreMatchers.containsString(urlSubstring)))`